host/src/proxy.rs

```diff
@@ -0,0 +1,21 @@
+use crate::WasiCtx;
+
+pub mod wasi {
+    wasmtime::component::bindgen!({
+        path: "../wit",
+        world: "proxy",
+        tracing: true,
+        async: true,
+    });
+}
+
+pub fn add_to_linker<T: Send>(
+    l: &mut wasmtime::component::Linker<T>,
+    f: impl (Fn(&mut T) -> &mut WasiCtx) + Copy + Send + Sync + 'static,
+) -> anyhow::Result<()> {
+    wasi::random::add_to_linker(l, f)?;
```

Comment:
I believe the poll and streams `add_to_linker` functions are missing here.

Reply:
Fixed.